ethcore/src/types/filter.rs

```diff
@@ -59,7 +65,8 @@ impl Clone for Filter {
 			from_block: self.from_block.clone(),
 			to_block: self.to_block.clone(),
 			address: self.address.clone(),
-			topics: topics[..].to_vec()
+			topics: topics[..].to_vec(),
```

Comment:
side note: seems that topic only has four members -- why bother allocating a Vec at all?